README.md

```diff
@@ -3,10 +3,26 @@
 [![Build Status](https://travis-ci.com/CalabreseLab/seekr.svg?branch=master)](https://travis-ci.com/CalabreseLab/seekr)
 [![Build Status](https://img.shields.io/pypi/v/seekr.svg)](https://pypi.python.org/pypi/seekr)
 
-Find communities of nucleotide sequences based on kmer frequencies.
+Find communities of nucleotide sequences based on *k*-mer frequencies.
 
 A web portal is available at [seekr.org](http://seekr.org).
 
+`The v1.0.1 update contains additional log transformation options and a tweak for length normalization`
+
+## Updates
+Includes a redesigned flag to indicate the method of *k*-mer standardization, and an additional option for *k*-mer standardization: --log2 [1,2,3] or -l [1,2,3]
```

Comment:
From a User Interface perspective, this command line option is no longer intuitive. 

Why are the choices named "1", "2" and "3"? Those numbers are completely arbitrary and don't give the user any indication of what they're choosing. It's a not too much different from picking "asxgg", "rrtn", and "7fm3" as the three choices. Both sets are meaningless.

But, you saved yourself with `Pass 1 for pre-standardization log transform, 2 post-standardization, 3 no log`.

Why don't we change the interface to be:

```
Pass "pre" for pre-standardization log transform, "post" for post-standardization, or "none" for no log
```

Then, maybe you can setup an `Enum` on the internal side of things:

https://docs.python.org/3/library/enum.html

 If you can't figure out enums, let me know, but it should be pretty straightforward. But this "1", "2" and "3" business is kinda spooky.

Reply:
Changing it now would involve re-writing code and the read-me, however it seems fairly straightforward. I will try but I don't have a lot of time